### ISSUE TITLE
Slider: Slider handle does not focus on interaction

### ISSUE THREAD

**Issue by ph-amogh** [primefaces/primereact#7927]
### Describe the bug

In primereact v10 on interaction with the slider handle via click or tab down, the slider handle does not move on arrow key presses nor does it have styling to indicate that it is in focus. However in v9 this works as expected.

This issue exists on the documentation page as well
https://primereact.org/slider/

### System Information

```Shell
System:
    OS: Linux 6.8 Ubuntu 22.04.5 2025.01.31 LTS (Cubic 2025-01-31 01:49) 22.04.5 LTS (Jammy Jellyfish)
    CPU: (12) x64 13th Gen Intel(R) Core(TM) i5-1345U
    Memory: 19.55 GB / 31.04 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 22.14.0 - ~/.nvm/versions/node/v22.14.0/bin/node
    npm: 11.2.0 - ~/.nvm/versions/node/v22.14.0/bin/npm
    pnpm: 10.7.0 - ~/.nvm/versions/node/v22.14.0/bin/pnpm
  Browsers:
    Chrome: 135.0.7049.84
  npmPackages:
    primereact: ^10.8.5 => 10.8.5 
    react: ^18.0.0 || ^17.0.0 => 18.3.1
```

### Steps to reproduce the behavior

Click on the slider handle or use tab to focus on to the component

### Expected behavior

Slider handle should be able to be controlled by arrow keys and have an outline on focus

**Comment by melloware** [comment]
Yep it looks like it is not adhering to its accessibility docs: https://primereact.org/slider/#accessibility

**Comment by codiini** [comment]
@melloware Has this been fixed? Seems to be working on my end.

**Comment by melloware** [comment]
@codiini i can close if it's working in 10.9.5!

**Comment by codiini** [comment]
@melloware Okay then, just test on your end to be sure and then I think this can be closed out.

**Comment by melloware** [comment]
Yep its all working for me in 10.9.5
